spec/presenters/tree_builder_ops_rbac_features_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe TreeBuilderOpsRbacFeatures do
+  let(:features) do
+    %w(
+      all_vm_rules
+      instance
+      instance_view
+      instance_show_list
+      instance_control
+      instance_scan
+    )
+  end
+
+  let(:role) do
+    FactoryGirl.create(:miq_user_role, :id => 100_000_002, :features => features)
+  end
+
+  let(:tree) do
+    TreeBuilderOpsRbacFeatures.new(
+      "features_tree",
+      "features",
+      {:trees => {}},
+      true,
+      :role     => role,
+      :editable => false
+    )
+  end
+
+  let(:main_keys) do
+    bs_tree.first["nodes"].map { |n| n['key'] }
+  end
+
+  describe 'bs_tree' do
+    subject(:bs_tree) { JSON.parse(tree.locals_for_render[:bs_tree]) }
```

Comment:
![what-kind-of-sorcery-is-this](https://cloud.githubusercontent.com/assets/649130/24403527/981fae76-13bd-11e7-8a88-f58b62688ec5.jpg)
Please use `let` when defining a lazy-loadable variable and `subject` without arguments when declaring the subject.